hyperon_das_atomdb/adapters/ram_only.py

```diff
@@ -572,31 +577,32 @@ def get_matched_links(
 
         patterns_matched = list(pattern) if (pattern := self.db.patterns.get(pattern_hash)) else []
 
-        if kwargs.get("toplevel_only"):
-            return kwargs.get("cursor"), self._filter_non_toplevel(patterns_matched)
+        if kwargs.get("toplevel_only", False):
+            return self._filter_non_toplevel(patterns_matched)
 
-        return kwargs.get("cursor"), patterns_matched
+        # return [handle for handle, _ in patterns_matched]
```

Comment:
should this be deleted?

Reply:
Also, any code related to `UNORDERED_LINK_TYPES` can be deleted.

Reply:
Fixed the former but not the later